src/igvfd/schemas/donor.json

```diff
@@ -67,6 +67,21 @@
             },
             "maxItems": 2
         },
+        "traits": {
+            "title": "Phenotypic traits",
+            "description": "A list of known phenotypic traits of the donor",
+            "type": "array",
+            "uniqueItems": true,
+            "default": [],
+            "minItems": 0,
+            "items": {
+                "title": "Phenotype term",
+                "description": "Phenotype term(s).",
```

Comment:
I don't understand why in the description of a single term you say "Phenotype term(s)." - I think it is singular.

Reply:
I tried to follow what we have done for other lists example https://github.com/IGVF-DACC/igvfd/blob/dev/src/igvfd/schemas/biosample.json#L281